bin/ModelValidator

```diff
@@ -23,17 +23,21 @@ class option_parser:
                                  metavar='schema',
                                  help='Schema name against the json will be validated'
                                  )
+        self.parser.add_argument('package',
+                                 metavar='package',
+                                 help='package of the schema'
+                                 )
         self.parser.add_argument('version',
                                  metavar='version',
                                  help='version of schema against the json will be validated'
                                  )
 
 
-def validation(json_file, schema, version):
+def validation(json_file, schema, version, package):
     if version == 'latest':
-        protocol = importlib.import_module('protocols.GelProtocols')
+        protocol = importlib.import_module('protocols.'+ package)
     else:
-        protocol = importlib.import_module('protocols.GelProtocols_' + version.replace('.', '_'))
+        protocol = importlib.import_module('protocols.' + 'package' + version.replace('.', '_'))
```

Comment:
Are we sure you want to use the string 'package' and not the variable package???